src/components/Footer.tsx

```diff
@@ -0,0 +1,26 @@
+
+import * as React from 'react';
+import './../styles.css';
+
+const styles: { [key: string]: React.CSSProperties } = {
```

Comment:
Would it be possible to keep the styling totally within the CSS file instead of having to import it? Could we use a container class on the div (or maybe .footer) and reference it in the JSX instead of writing and passing it in as an attribute?